app/executor.php

```diff
@@ -0,0 +1,802 @@
+<?php
```

Comment:
Not sure i understand

Reply:
in the PR this file is shown as completely new file being added, probably because this was removed from `appwrite/appwrite` and moved to `open-runtimes` please consult @Meldiron

Reply:
removed